### ISSUE TITLE
Expand scope to include how software is provisioned?

### ISSUE THREAD

**Issue by willfurnass** [mikecroucher/HPC_schedulers#4]
Could expand schema to include:
 - how centrally-provided software built and provisioned ('artisan' builds, EasyBuild, Spack, Alces, Nix, container registry, conda)
 - how software loaded (Environment Modules, Lmod, `spack load`/spack environments, conda environments, containery thing pull/run/exec, CVMFS)

**Comment by mikecroucher** [comment]
It's a good idea but how easy is it to get such info from public information?

**Comment by takluyver** [comment]
I've only got experience of one HPC cluster, but in that one, it's not possible to answer these as clearly as with the scheduling system. Different bits of software are managed in different ways, and in some cases by different groups of people.

**Comment by willfurnass** [comment]
Yes, probably difficult to get info on this. The answers would be interesting though.
